### ISSUE TITLE
configgen layout

### ISSUE THREAD

**Issue by Cissoubaka** [digitalLumberjack/recalbox-os#427]
Add a setting to configgen to handle the layout, 
exemple : neogeo.layout = /path/to/file/neogeo-layout.cfg 
and this file only contains the buttons mapping for the neogeo

**Comment by digitalLumberjack** [comment]
Good idea.

**Comment by substring** [comment]
that can also be done in core remaps, that's how i did it for neogeo. Have you tried @Cissoubaka ? Do you think it fits your need ?

**Comment by Cissoubaka** [comment]
yeah it's working, but it's less flexible than a line in recalbox.conf
